src/solvers/dg_structured/containers_3d.jl

```diff
@@ -0,0 +1,68 @@
+function init_elements!(elements, mesh::StructuredMesh{3, RealT}, nodes) where {RealT}
+  n_nodes = length(nodes)
+
+  @unpack coordinates_min, coordinates_max, linear_indices = mesh
+
+  # Get cell length
+  dx = (coordinates_max[1] - coordinates_min[1]) / size(mesh, 1)
+  dy = (coordinates_max[2] - coordinates_min[2]) / size(mesh, 2)
+  dz = (coordinates_max[3] - coordinates_min[3]) / size(mesh, 3)
+
+  inverse_jacobian = 8/(dx * dy * dz)
+
+  # Calculate inverse Jacobian and node coordinates
+  for element_x in 1:size(mesh, 1), element_y in 1:size(mesh, 2), element_z in 1:size(mesh, 3)
```

Comment:
I think this is a slight misnomer: `element_x` should really be `cell_x`, since you are iterating over the *cells* of the mesh, not the elements - or am I missing something?

Reply:
Maybe something like in 1decca70 (feel free to rewrite/amend)?